gateway/src/apicast/executor.lua

```diff
@@ -58,4 +59,11 @@ function _M:context(phase)
     return shared_build_context(self)
 end
 
+local metrics = _M.metrics
+--- Render metrics from all policies.
+function _M:metrics(...)
```

Comment:
Maybe we should add a test for this. Define 2 policies with only a metrics phase that incr some counter and make sure that this returns the sum, or something similar.

Reply:
Yeah but this does not really "return" anything. It renders the output.
I'll definitely do that when moving some policies with metrics to upstream from https://github.com/3scale/apicast-cloud-hosted/pull/5.